javascripts/discourse/components/stat-banner.gjs

```diff
@@ -44,7 +44,7 @@ export default class StatBanner extends Component {
   get shouldShow() {
     if (
       !settings.display_stats ||
-      (settings.hide_on_mobile && this.site.mobile_view)
+      (settings.hide_on_mobile && this.site.mobileView)
```

Comment:
We now have support for system tests and acceptance tests. I think a test here will help us in the long run.